core/schema/file.go

```diff
@@ -50,6 +50,25 @@ func (s *fileSchema) Install() {
 			Args(
 				dagql.Arg("name").Doc(`Name to set file to.`),
 			),
+		dagql.NodeFunc("search", DagOpWrapper(s.srv, s.search)).
+			View(AllVersion). // entries returns different results in different versions
+			Doc(`Searches for content matching the given pattern, which may be a regular expression or a literal string.`).
+			Args(
+				dagql.Arg("pattern").Doc(`The text to match.`),
+				dagql.Arg("literal").Doc(`Interpret the pattern as a literal string instead of a regular expression.`),
+				dagql.Arg("multiline").Doc(`Enable searching across multiple lines.`),
+			),
+		dagql.NodeFunc("withReplaced",
```

Comment:
A bit less convinced on this particular API, but no real objections, it feels useful :tada:

But I think it should support the same args as `search` (e.g. `literal`, `multiline`)?

Kinda wish we supported multiple return values :cry: It would be cool to support returning what actually got replaced.